hs628_tools/feature_missing.py

```diff
@@ -0,0 +1,201 @@
+import numpy as np
+from numpy.core.multiarray import ndarray
+from sklearn.model_selection import train_test_split
+from sklearn.ensemble import RandomForestRegressor, RandomForestClassifier
+from sklearn.model_selection import StratifiedKFold
+
+
+class FeatureRemover:
+    """This Class is created to perform feature removal tools for data cleaning.
+
+    #There are four methods in this class:
+
+        1. which_missing: Returns columns with a missing percentage greater than a specified threshold 2.
+        single_value: Returns columns with a single unique value 3. colinear: Returns collinear columns with a
+        correlation greater than a specified correlation threshold 4. low_importance: Returns features that are
+        ranked low importance in the Random Forest Regressor or Classifier #Notes: using the idea adapted from:
```

Comment:
@EHWUSF My code is entirely different from the source I mentioned here since I'm using numpy arrays instead of Pandas. 
Would you recommend changing the language from "using the idea adapted from X" to "adapted from X" ?